### PR TITLE
[BPK-4246]: Button radius updates

### PR DESCRIPTION
**Pull request by olliecurtis**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,12 +83,13 @@
     "babel-loader": "^8.0.6",
     "babel-plugin-jest-hoist": "^26.2.0",
     "backpack-react-native": "file:./lib",
+    "bpk-svgs": "^12.1.16",
```
Comment (olliecurtis) [inline package.json:86]: Added this in the root as a devDependency as it is not getting installed through `lib/` which then causes iOS CI and local to fail as the module is not installed which is required to copy the svg font files over from the npm package to the iOS build folder

Comment (georgegillams) [reply package.json:86]: Is this hack needed in CI, or just locally?

Comment (olliecurtis) [reply package.json:86]: Both

**Review by georgegillams — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (olliecurtis) [reply package.json:86]: I guess this is something to explore as part of the ticket we have to investigate running locally

**Pull request merged**